packages/ibm-products/src/components/BigNumbers/BigNumbers.stories.jsx

```diff
@@ -32,44 +34,37 @@ const numericOptions = {
   'undefined ': undefined,
 };
 
+const iconButtonOptions = {
```

Comment:
Added the ability to show/hide the Edit button via a Storybook dropdown. This means we could remove the `withEditButton` page.